Formula/openshift-cli.rb

```diff
@@ -1,16 +1,14 @@
 class OpenshiftCli < Formula
   desc "OpenShift command-line interface tools"
   homepage "https://www.openshift.com/"
-  url "https://github.com/openshift/oc.git",
-      tag:      "openshift-clients-4.12.0-202208031327",
-      revision: "3c85519af6c4979c02ebb1886f45b366bbccbf55"
+  url "https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.12.15/openshift-client-src.tar.gz"
```

Comment:
```suggestion
  url "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.12.15/openshift-client-src.tar.gz"
```

Seems like the URL without `x86_64` works?

Reply:
Good catch! Updated.